### PR TITLE
[FIX] rma_sale: portal permissions

### PR DESCRIPTION
**Pull request by chienandalu**
Some operations need to be sudoed to be reachable by the portal user

cc @Tecnativa TT24986

### CODE REVIEW

**Comment by chienandalu** [comment]
ping @pedrobaeza @ernestotejeda 
Sorry about the double PR

File: rma_sale/controllers/sale_portal.py [hunk 1]
```diff
@@ -25,12 +25,13 @@ def request_rma(self, order_id, access_token=None, **post):
...
         # Create wizard an generate rmas
-        location_id = order_obj.browse(order_id).warehouse_id.rma_loc_id.id
+        order = order_obj.browse(order_id).sudo()
+        location_id = order.warehouse_id.rma_loc_id.id
         wizard = wizard_obj.with_context(active_id=order_id).create({
             'line_ids': line_vals,
             'location_id': location_id
         })
-        rma = wizard.create_rma(from_portal=True)
+        rma = wizard.sudo().create_rma(from_portal=True)
```
Comment (pedrobaeza) [inline rma_sale/controllers/sale_portal.py:34]: The problem with this is that the create user will be system, not the proper user. Fortunately, this won't happen in v13, right?

**Review by pedrobaeza — APPROVED** [review]

File: rma_sale/controllers/sale_portal.py (hunk 1 above)
Comment (chienandalu) [reply rma_sale/controllers/sale_portal.py:34]: > Fortunately, this won't happen in v13, right?

Why not?

Comment (pedrobaeza) [reply rma_sale/controllers/sale_portal.py:34]: `sudo()` only change the permissions, not the uid.

Comment (chienandalu) [reply rma_sale/controllers/sale_portal.py:34]: :+1:

**Pull request merged**